### PR TITLE
v2.3.13

### PR DESCRIPTION
**Pull request by islandbitcoin**
Migration to PostgreSQL DB and version bump

### CODE REVIEW

**Comment by BitcoinMechanic** [comment]
This fixed an issue for a user I had with database errors as expected.

File: Makefile [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
+	rm -f scripts/*.js
+
+scripts/embassy.js: $(TS_FILES)
+	deno bundle scripts/embassy.ts scripts/embassy.js
 
-image.tar: Dockerfile docker_entrypoint.sh health-check.sh
-	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --tag start9/photoview/main:${EMVER} --platform=linux/arm64/v8 -o type=docker,dest=image.tar .
\ No newline at end of file
+image.tar: Dockerfile docker_entrypoint.sh health-check.sh example.env
+	docker buildx build --tag start9/$(PKG_ID)/main:$(PKG_VERSION) --platform=linux/arm64 -o type=docker,dest=image.tar .
```
Comment (chrisguida) [inline Makefile:25]: Why remove `DOCKER_CLI_EXPERIMENTAL=enabled`? Is this no longer needed?

File: docker_entrypoint.sh [hunk 2]
```diff
@@ -36,38 +51,71 @@ if [ -z "$USERS" ]; then
...
+#   sed -i '/^$/d' /media/photoview.sql
+#   sed -i "s/(1/(10/" /media/photoview.sql
+#   touch /media/photoview_trimmed.sql
+#   tail -n+3 /media/photoview.sql >> /media/photoview_trimmed.sql
+#   su - postgres -c "psql -d photoview < /media/photoview_trimmed.sql"
+#   echo 'updating admin password...'
+#   export OLD_PASSWORD=$(su - postgres -c 'psql -d '$POSTGRES_DB' -c "select password from users limit 1"')
+#   sed -i "s/$POSTGRES_PASSWORD/$OLD_PASSWORD/" /media/start9/stats.yaml 
+#   export POSTGRES_PASSWORD=$OLD_PASSWORD
+# fi
```
Comment (chrisguida) [inline docker_entrypoint.sh:105]: What's this comment block for? Are we migrating old data somewhere else?

File: manifest.yaml [hunk 3]
```diff
@@ -35,10 +39,13 @@ migrations:
       entrypoint: migration-from-lt-2-3-9.sh
       args: []
       io-format: json
-      inject: true
+      inject: false
```
Comment (chrisguida) [inline manifest.yaml:42]: What's the reason for this change?

**Comment by BitcoinMechanic** [comment]
I should add - the password for PV got changed during this upgrade - that's probably not intentional.

File: Makefile (hunk 1 above)
Comment (islandbitcoin) [reply Makefile:25]: > Why remove `DOCKER_CLI_EXPERIMENTAL=enabled`? Is this no longer needed?

updated to use the new Makefile format from hello-world example.

File: docker_entrypoint.sh (hunk 2 above)
Comment (islandbitcoin) [reply docker_entrypoint.sh:105]: removed

File: manifest.yaml (hunk 3 above)
Comment (islandbitcoin) [reply manifest.yaml:42]: was throwing an error. I'm assuming this is because of the OS updates

**Comment by islandbitcoin** [comment]
> I should add - the password for PV got changed during this upgrade - that's probably not intentional.

this is very intentional. the old password formats do not migrate properly.

File: Makefile (hunk 1 above)
Comment (chrisguida) [reply Makefile:25]: ...I assume the new hello-world was tested and works, and doesn't require this flag anymore?

File: manifest.yaml (hunk 3 above)
Comment (chrisguida) [reply manifest.yaml:42]: Did you test? I seriously doubt this was the correct solution

Comment (islandbitcoin) [reply manifest.yaml:42]: I cannot test it because there is not a version earlier than 2.3.9 on the marketplace

File: Makefile (hunk 1 above)
Comment (islandbitcoin) [reply Makefile:25]: seems that way

File: manifest.yaml (hunk 3 above)
Comment (chrisguida) [reply manifest.yaml:42]: That means that that migration is for people updating from 0.2.x. We'll have to test that so users can migrate from 0.2.x.

Comment (chrisguida) [reply manifest.yaml:42]: We have since started catching this misconfiguration: https://github.com/Start9Labs/embassy-os/pull/1653

**Comment by islandbitcoin** [comment]
Beta testing is well underway, I think this code is ready to be merged? @chrisguida
